### DISCUSSION TITLE
Download only specific multiplatform bundle

### DISCUSSION THREAD

**Discussion by GazziFX** [SmartlyDressedGames/Unturned-3.x-Community#3665] [Ideas]
Is there any way to download bundles from workshop only for current platform? Or include only shaders for different bundles

**Comment by SDGNelson** [comment]
Unfortunately this is not supported by Steam Workshop AFAIK.

**Reply by GazziFX** [reply]
What about creating single bundle for textures, prefabs etc and separate bundles for shaders?

**Comment by educatalan02** [comment]
All the bugs the unturned mods contains are related to Steam Workshop. Maybe that is a thing to take a look for the long term.

**Reply by educatalan02** [reply]
https://user-images.githubusercontent.com/34221626/215431218-19507b99-eff5-4ac2-94ed-4d5995269597.mp4

That's the perfect example of how Steam Workshop works.
I've been trying to download a mod like +40 mins...
Verify doesn't work, Clearing Downloads cache from Steam doesn't work & Restarting Steam doesn't too.

To be clear, imagine usual players when they join a modded server. They usually get kicked "missing asset" or similar errors. And this errors aren't fixable by "Reinstalling the game nor verifying files"

**Reply by GazziFX** [reply]
Maybe create server property for mods repository link, and they will be downloaded from that specific web server in zip
